setup.py

```diff
@@ -24,17 +25,26 @@ def get_version():
     raise RuntimeError('Unable to find version string in %s.' % (file_path,))
 
 
-def load_requirements(*requirements_paths):
+def load_requirements(requirements_folder):
     """
     Load all requirements from the specified requirements files.
     Returns a list of requirement strings.
     """
+    edx_platform_version = get_edx_platform_version()
+
+    if edx_platform_version == 0.9:
+        requirements_path = '{}/{}'.format(requirements_folder, 'ginkgo.in')
+    elif edx_platform_version == 0.10:
+        requirements_path = '{}/{}'.format(requirements_folder, 'hawthorn.in')
+    else:
+        requirements_path = '{}/{}'.format(requirements_folder, 'ironwood.in')
+
```

Comment:
We must handle a default case, if the edx_platform_version doesn't match with any of the options. @andrey-canon

Reply:
ironwood is the default case, this plugin has not support for previous version to ginkgo and later version to ironwood.

Reply:
The default case must be an error, since if there is not recognized edX-platform version, we cannot support that. @andrey-canon 
We need a specific platform version otherwise, we should not let it to install it.